src/attribution/onLCP.ts

```diff
@@ -65,7 +71,7 @@ const attributeLCP = (metric: LCPMetric): LCPMetricWithAttribution => {
       );
 
       attribution = {
-        element: getSelector(lcpEntry.element),
+        element: getSelector(lcpEntry.element || lcpTargetMap.get(lcpEntry)),
```

Comment:
```suggestion
        element: getSelector(lcpEntry.element ?? lcpTargetMap.get(lcpEntry)),
```
I don't think this will ever make a difference, but may as well use the more appropriate operator for the job.

Reply:
I presume it actually makes sense to do this rather than just `lcpTargetMap.get(lcpEntry)` all the time? Would be smaller code to just do that and in theory it should be the same thing but guess the `lcpEntry.element` might be slightly faster? WDYT?

Reply:
I think this is fine, though I can't think of any cases where just using `lcpTargetMap.get()` would return anything different...